interproscan/5.72-103.0/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM mambaorg/micromamba:2.0.5
+
+ARG VERSION=5.72-103.0
+
+LABEL software="InterProScan"
+LABEL software.version="5.72-103.0"
+LABEL description="InterProScan is a search engine for InterPro is a database which integrates together predictive information about proteins' function."
+LABEL website="https://github.com/ebi-pf-team/interproscan"
+LABEL license="Apache 2.0"
+
+COPY --chown=$MAMBA_USER:$MAMBA_USER env.yml /tmp/env.yml
+
+RUN micromamba install -y -n base -f /tmp/env.yml \
+    && micromamba install -y -n base conda-forge::procps-ng \
```

Comment:
Any reason this shouldn't just be in the env.yml?

Reply:
hehe.. not really.. that was just me copying and pasting from s. containers.

Reply:
I'll keep it this way as this is how s.containers does it (there must be a reason for it). I'll investigate when we tackle the whole containers thing we disucssed